examples/tstore_bug-2/main.go

```diff
@@ -38,41 +39,40 @@ func makeTest() error {
 	gst := fuzzing.BasicStateTest("Cancun")
 
 	a := common.HexToAddress("0xaa")
-	b := common.HexToAddress("0xbb")
 
-	// 0xaa calls 0xbb, then checks TLOAD(4) and puts it into state
+	// bb is the initcode of the contract to be created
 	{
-		aa := program.NewProgram()
-		aa.CallCode(nil, b, nil, 0, 0, 0, 0)
+		bb := program.NewProgram()
 
-		aa.Push(4)
-		aa.Op(ops.TLOAD)
+		// Calculate code length based on the output of TLOAD
+		// If TLOAD results in 0, the code length is 10
+		// For anything else, it is too large
+		bb.Push(1)
+		bb.Op(ops.TLOAD)
 
-		aa.Push(1)
-		aa.Op(ops.SSTORE)
+		bb.Push(1)
+		bb.Push(1)
+		bb.Op(ops.TSTORE)
 
-		gst.AddAccount(a, fuzzing.GenesisAccount{
-			Code:    aa.Bytecode(),
-			Balance: big.NewInt(0),
-			Storage: make(map[common.Hash]common.Hash),
-		})
-	}
-
-	// 0xbb does a TSTORE, then exits on revert
-	{
+		bb.Op(ops.ISZERO)
+		bb.Push(24576) // MAX_CODE_SIZE 0x6000
+		bb.Op(ops.MUL)
```

Comment:
Interesting. This looks like a hack to avoid jumping, is that so? Instead of branching on `1/0`, you multiply it by `0x6000` and add `10`?

Reply:
Yeah. Still figuring out `JumpIf` in `goevmlab` but this effectively accomplishes what I intended to do in the init code :)

Reply:
Nah jumpif isn't any good for this. Backward jumps are easy, forward jumps are a bit more tedious in goevmlab. It's not a proper "compiler", just some tooling to construct evm bytecode very minimalistically. 

What one can do is basically note the size, add the jump to `0`, yada yada, then later on, replace the `0`-byte with the actual destinatoin. 

I made a different/simpler fix though, returning code of size `0x600a - tstore(1)`